server-x/README.md

```diff
@@ -0,0 +1,86 @@
+# Keycloak.X Docker Image
+
+The `Dockerfile` for Keycloak.X Docker Image.
+
+See the `Build` section for more details on how to build the image.
+
+Once built, you can run the server in the same manner as when using the Keycloak.X distribution by passing any command-line argument:
+
+    docker run --name keycloak -p 8080:8080 \
+           -e KEYCLOAK_ADMIN=admin -e KEYCLOAK_ADMIN_PASSWORD=change_me \
+           IMAGE[:TAG] \
+           start-dev
+           
+## Extending the Image
+
+To customize the base image, create a new `Dockerfile` similar to following:
+
+```
+FROM quay.io/keycloak/keycloak-x
+
+WORKDIR /opt/jboss/keycloak
+RUN keytool -genkeypair -storepass password -storetype PKCS12 -keyalg RSA -keysize 2048 -dname "CN=server" -alias server -ext "SAN:c=DNS:localhost,IP:127.0.0.1" -keystore conf/server.keystore
+```
+
+In the example above, you are basically creating a new image `FROM` this image and adding a Java Keystore to configure HTTPS using a self-signed certificate (only for example purposes, never do that for production).
+
+As another example, you can install any custom provider you may have (include themes within a JAR file) as follows:
+
+```
+FROM pedroigor/keycloak-x
+
+COPY my-providers/ /opt/jboss/keycloak/providers/
+
+WORKDIR /opt/jboss/keycloak
+
+RUN keytool -genkeypair -storepass password -storetype PKCS12 -keyalg RSA -keysize 2048 -dname "CN=server" -alias server -ext "SAN:c=DNS:localhost,IP:127.0.0.1" -keystore conf/server.keystore
+
+# Run the config command to install custom providers
+RUN ./bin/kc.sh config
+```
+
+In the example above, any JAR files within the directory `my-providers` will be copied to the image's `/opt/jboss/keycloak/providers` directory. Then the `config` command is executed to install the custom providers.
+
+For last, you can configure the server using any configuration option available as follows:
+
+```
+FROM pedroigor/keycloak-x
+
+COPY my-providers/ /opt/jboss/keycloak/providers/
+
+WORKDIR /opt/jboss/keycloak
+
+RUN keytool -genkeypair -storepass password -storetype PKCS12 -keyalg RSA -keysize 2048 -dname "CN=server" -alias server -ext "SAN:c=DNS:localhost,IP:127.0.0.1" -keystore conf/server.keystore
+
+RUN ./bin/kc.sh config --db=mariadb --db-url=jdbc:mariadb://192.168.0.15/keycloak --db-username=keycloak --db-password=password
+```
+
+In the example above, the configuration is being set to use a PostgreSQL database.
```

Comment:
You meant MariaDB instead of PostgreSQL here? Or alternative is to change the command to use PostgreSQL database.